configs/ttalps_histogrammer_config.py

```diff
@@ -79,7 +79,7 @@
 muonVertexCollections = {
     "GoodPFIsoDimuonVertex": muonVertexBaselineSelection + ["PFRelIsolationCut"],
     "BestPFIsoDimuonVertex": muonVertexBaselineSelection + ["PFRelIsolationCut", "BestDimuonVertex"],
-    "BestDimuonVertex": muonVertexBaselineSelection + ["BestDimuonVertex"],
+    # "BestDimuonVertex": muonVertexBaselineSelection + ["BestDimuonVertex"],
```

Comment:
I think we should keep this uncommented - for J/Ψ, this is the dimuon we want to look at. It doesn't really hurt to fill these histograms for other regions, either. One thing I never look at is GoodPFIsoDimuonVertex - if you don't use them anymore, maybe this is something we could get rid of.